README.md

```diff
@@ -1,7 +1,89 @@
-# Organic Platform
-Wordpress Plugin
+# Organic Wordpress Plugin
+https://wordpress.org/plugins/organic/
 
-## Configuration
+# Development
+## Initial setup
+1. [Install poetry](https://python-poetry.org/docs/#installation)
+2. Make sure poetry will use your currently active Python
+``` bash
+$ poetry config virtualenvs.prefer-active-python true
+```
+
+3. Install poetry plugins
+```bash
+$ poetry self add poetry-dotenv-plugin
+$ poetry self add poetry-pre-commit-plugin
+```
+
+4. (Optional) Before setting up project you may want [to install pyenv](https://github.com/pyenv/pyenv#installation) and use it [to configure the latest Python](https://python-poetry.org/docs/managing-environments/)
+```bash
+$ pyenv install 3.11
+$ pyenv local 3.11
+```
+
+5. Install deps
+```bash
+$ peotry install
+```
+
+## Work on plugin
+1. Build the dev environment
+```bash
+$ poetry run ./dev.py up
+```
+
+2. Lint files
+```bash
+$ poetry run ./dev.py lint
+```
+
+3. Shutdown dev enviroment
+```bash
+$ poetry run ./dev.py down
+```
+
+For more command and helpful flags check the help
+```bash
+$ poetry run ./dev.py --help
+$ poetry run ./dev.py up --help
+$ poetry run ./dev.py down --help
+```
+
+## Organic Affiliate Features Development
+The Wordpress Plugin includes the following Affiliate App features:
+* Insert Product Card: implemented as a [Gutenberg Block](https://developer.wordpress.org/block-editor/getting-started/create-block/).
+* Insert Product Carousel: implemented as a [Gutenberg Block](https://developer.wordpress.org/block-editor/getting-started/create-block/).
+* Insert Affiliate Link: implemented as a [Custom Format](https://developer.wordpress.org/block-editor/how-to-guides/format-api/).
+
+These blocks are atomatically build on every file change after `./dev.py up` - see `docker compose logs -f --tail 20 nodejs` for logs
```

Comment:
Super minor, but when we have time, just two small typos: "These blocks are automatically built on every file change after"